### PR TITLE
WIP: Ansible Content Collection specification

### PR DESCRIPTION
**Pull request by cutwater**

### CODE REVIEW

File: docs/docsite/rst/mazer/collection.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+in `dependencies` section as a mapping between collection name and a
+version string. The version string is a semver version spec:
+
+.. code-block:: yaml
+
+  dependencies:
+    geerlinguy.java: "1.3.0"
+
+Content
+^^^^^^^
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/collection.rst]: This is not necessary for the first iteration. It over complicates the build process and testing. Please remove this for the MVP.

File: docs/docsite/rst/mazer/collection.rst [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    - "Bob Smith (http://bsmith.example.com)"
+    - "Jack Jackson"
+
+`description`
+^^^^^^^^^^^^^
+
+A short summary description of the collection.
+
+`license`
+^^^^^^^^^
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/collection.rst]: SPDX value is required.

File: docs/docsite/rst/mazer/collection.rst [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  license: "MIT"
+
+You can use `The SPDX License List <https://spdx.org/licenses/>`_.
+`OSI Approvded <https://opensource.org/licenses/alphabetical>`_ licenses
+are more preferrable.
+
+keywords
+^^^^^^^^
+
+A list of collection keywords.
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/collection.rst]: Should follow the same criteria as Galaxy tags, where a tag can only contain alphanumeric characters.

File: docs/docsite/rst/mazer/collection.rst [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+you are using:
+
+.. code-block:: yaml
+
+  license: "MIT"
+
+You can use `The SPDX License List <https://spdx.org/licenses/>`_.
+`OSI Approvded <https://opensource.org/licenses/alphabetical>`_ licenses
+are more preferrable.
+
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/collection.rst]: Missing 'license_file'. Below, in the example manifest.json, their is a 'license_file' key.

File: docs/docsite/rst/mazer/collection.rst [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+    "license_file": null,
+    "readme": "README.md",
+
+    "dependencies": {
+      "geerlinguy.java": {"version": "1.3.0"},
+      "geerlinguy.nginx": {
+        "git": "http://github.com/geerlinguy/ansible-role-nginx.git",
+        "branch": null
+      }
+    },
```
Comment (chouseknecht) [inline docs/docsite/rst/mazer/collection.rst]: The above block should be formatted like this:

```
"collection_info": {
        "description": "testing some foo",
        "license": "GPL-3.0-or-later",
        "readme": "README.md",
        "version": "0.0.1",
        "dependencies": [],
        "authors": [
            "chouseknecht"
        ],
        "keywords": [
            "foo",
            "bar",
            "baz"
        ],
        "name": "foo.foo"
    },
    "format_version": 0.0
```

Comment (chouseknecht) [reply docs/docsite/rst/mazer/collection.rst]: Also, please change '_format' to 'format', as the '_' represents a private attribute that can't be set in a Python model, when using `attrs`

Comment (cutwater) [reply docs/docsite/rst/mazer/collection.rst]: I renamed `_format` to `format`. Keeping MANIFEST structure as is, as a result of today's call.

File: docs/docsite/rst/mazer/collection.rst (hunk 4 above)
Comment (cutwater) [reply docs/docsite/rst/mazer/collection.rst]: Done.

File: docs/docsite/rst/mazer/collection.rst (hunk 3 above)
Comment (cutwater) [reply docs/docsite/rst/mazer/collection.rst]: Done.

File: docs/docsite/rst/mazer/collection.rst (hunk 1 above)
Comment (cutwater) [reply docs/docsite/rst/mazer/collection.rst]: Done.

**Comment by chouseknecht** [comment]
Should we require:
- An `scm_url`, or something that captures a link to the public source code. I presume we would want to require the source to be public.
- A link to public issue tracker

**Comment by cutwater** [comment]
> Should we require:
> 
> * An `scm_url`, or something that captures a link to the public source code. I presume we would want to require the source to be public.
> * A link to public issue tracker

Yes, this is good idea. I will add urls for source code, issue tracker, documentation, etc.
Do we really require the source to be public? We should have ToC then.

**Pull request closed without merging**